remove hypconfigmgmt install from Vagrantfile

vdloo opened the pull request:
Automatically installing the correct bundled plugin no longer works since [Augustus 31 2018](https://stackoverflow.com/a/28801317)

vdloo committed:
remove hypconfigmgmt install from Vagrantfile

Darkbelg commented:
@vdloo Can't you use [vagrant settings](https://www.vagrantup.com/docs/vagrantfile/vagrant_settings.html) to check if plugins are installed ?

vdloo committed:
use config.vagrant.plugins to install plugin

> Can't you use vagrant settings to check if plugins are installed ?

Thanks @darkbelg didn't know of that feature. That didn't exist when I
originally added the automatic install in the Vagrantfile.

vdloo commented:
Great @Darkbelg! I did not know that existed. I've added that to the Vagrantfile. Does this branch work for you? I have no ubuntu bionic environment I can test the full thing on at the moment unfortunately.

Darkbelg commented:
I don't know if this is bad or good news. But they fixed the bug in 2.1.5 and now the normal Vagrantfile is working again .
https://github.com/hashicorp/vagrant/blob/v2.1.5/CHANGELOG.md
> core: Fix Vagrant.has_plugin? behavior before plugins are initialized [GH-10165]

So I don't know if you wan't to continue with this change or not. But, I have tested it locally on vagrant 2.1.5 and it works. Why not include vagrant-hostmanager as a requirement ?
